plugins/workspace-minimap/src/positioned_minimap.ts

```diff
@@ -179,19 +179,23 @@ export class PositionedMinimap
    */
   private setAttributes(): void {
     const injectDiv = this.minimapWorkspace.getInjectionDiv();
-    const style = injectDiv.parentElement.style;
-    style.zIndex = '2';
-    style.position = 'absolute';
-    style.width = `${this.width}px`;
-    style.height = `${this.height}px`;
-    style.top = `${this.top}px`;
-    style.left = `${this.left}px`;
-    Blockly.svgResize(this.minimapWorkspace);
+    if(injectDiv.parentElement === null) {
+      return;
+    } else {
```

Comment:
No need to wrap in an `else` if you are early returning. Please remove the `else` part of this clause.